guides/common/assembly_using-external-databases.adoc

```diff
@@ -15,7 +15,7 @@ endif::[]
 To create and use external databases for {Project}, you must complete the following procedures:
 
 . xref:preparing-a-host-for-external-databases_{context}[].
-Prepare a {RHEL} 7 server to host the external databases.
+Prepare a {EL} 8 or {EL} 7 server to host the external databases.
```

Comment:
Please make your change ifndef orcharhino only. (plus the change from rhel to el.

Reply:
Excluded orcharhino. :+1: 
I guess RHEL was already replaced with EL.

Reply:
I meant something like this:

````diff
diff --git a/guides/common/assembly_using-external-databases.adoc b/guides/common/assembly_using-external-databases.adoc
index 0cb649dfd..31a25f103 100644
--- a/guides/common/assembly_using-external-databases.adoc
+++ b/guides/common/assembly_using-external-databases.adoc
@@ -14,8 +14,14 @@ endif::[]

 To create and use external databases for {Project}, you must complete the following procedures:

+ifdef::orcharhino[]
 . xref:preparing-a-host-for-external-databases_{context}[].
-Prepare a {RHEL} 8 server to host the external databases.
+Prepare a {RHEL} 7 server to host the external databases.
+endif::[]
+ifndef::orcharhino[]
+. xref:preparing-a-host-for-external-databases_{context}[].
+Prepare a {EL} 8 or {EL} 7 server to host the external databases.
+endif::[]
 . xref:installing-postgresql_{context}[].
 Prepare PostgreSQL with databases for {Project}, Candlepin and Pulp with dedicated users owning them.
 . xref:Configuring_Server_to_Use_External_Databases_{context}[].
````

Reply:
Addressed. I hope, I understood better.